interface/forms/aftercare_plan/new.php

```diff
@@ -26,153 +28,198 @@
-
-
-    <tr>
-
-        <td class="forms-subtitle" colspan="4"><B><?php echo xlt('Goal B');?>:</B>&nbsp;<?php  echo xlt('Emotional / Behavioral Conditions & Complications'); ?></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">1.</td>
-        <td colspan="3"><textarea name="goal_b_emotional_behavioral_conditions" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_b_emotional_behavioral_conditions"]);?></textarea></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">2.</td>
-        <td colspan="3"><textarea name="goal_b_emotional_behavioral_conditions_I" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_b_emotional_behavioral_conditions_I"]);?></textarea></td>
-
-    </tr>
-
-
-        <td class="forms-subtitle" colspan="4"><B><?php echo xlt('Goal C');?>:</B>&nbsp;<?php  echo xlt('Relapse Potential'); ?></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">1.</td>
-        <td colspan="3"><textarea name="goal_c_relapse_potential" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_c_relapse_potential"]);?></textarea></td>
-
-    </tr>
-    <tr>
-        <td align="right" class="forms">2.</td>
-        <td colspan="3"><textarea name="goal_c_relapse_potential_I" rows="2" cols="80" wrap="virtual name"><?php echo text($obj["goal_c_relapse_potential_I"]);?></textarea></td>
-
-    </tr>
-
-    <tr>
-        <td align="left colspan="3" style="padding-bottom:7px;"></td>
-    </tr>
-    <tr>
-        <td></td>
-    <td><input type='submit' value='<?php echo xla('Save'); ?>' class='button-css' />&nbsp;
-    <input type='button' value='<?php echo xla('Print'); ?>' id='printbutton' class='button-css' />&nbsp;
-    <input type='button' class='button-css' value='<?php echo xla('Cancel'); ?>'
- onclick="parent.closeTab(window.name, false)" /></td>
-    </tr>
-</table>
-</form>
+<div class="container">
+    <div class="row">
+        <div class="mt-5">
+            <div class="col-md-12 mt-5">
+                <h1 class="forms-title"><?php echo xlt('After Care Planning'); ?></h1>
+            </div>
+        </div>
+        <div class="col-md-12 mt-4">
+            <?php
+            echo "<form method='post' name='my_form' " .
+                "action='$rootdir/forms/aftercare_plan/save.php?id=" . attr_url($formid) . "'>\n";
+            ?>
+            <input type="hidden" name="csrf_token_form" value="<?php echo attr(CsrfUtils::collectCsrfToken()); ?>" />
+
+            <table class="table">
+                <tr>
+                    <td  class="forms" class="forms">
+                        <?php echo xlt('Client Name'); ?>:
+                    </td>
+                    <td class="forms">
+                        <label class="forms-data">
+                            <?php if (is_numeric($pid)) {
```

Comment:
I know it's a hold over from the previous form, but PID is likely always numeric, and if not, things will break way before this. Can probably remove this conditional